md/assets.md

```diff
@@ -72,9 +72,14 @@ A page can not have more than one localization asset. When you upload a new `loc
 
 The list contains both page assets and the assets of the custom widgets used in this page.
 
-In the list, you can delete page assets only, view local assets, and edit external assets of a page. You can not manually change the order of assets in the page. You can also indicate whether an asset is active. To avoid dependency clashes with multiple version of assets, make sure only one version of an asset is active in a page.
+In the list, you can delete page assets only, view local assets, and edit external assets of a page. You can also indicate whether an asset is active. To avoid dependency clashes with multiple version of assets, make sure only one version of an asset is active in a page.
+
+You can manually change the order of CSS and Javascript page assets. Reorder this files can be useful to override some css classes or javascript functions.
+
+At runtime, custom widget assets are loaded in alphabetical order of the custom widgets. For each custom widget, the assets are loaded in the order they are listed. 
+
+Then page assets are loaded. JavaScript and CSS assets are loaded in the page header, this loading is done as it's represented in this list, CSS first then Javascript.
```

Comment:
"this loading is done as it's represented in this list" 
==> "this loading is done according the list order"